### ISSUE TITLE
Server-side support?

### ISSUE THREAD

**Issue by enenbee** [enjarai/do-a-barrel-roll#25]
This would be probably way more difficult to implement actually then I think, but it would be useful if there could be a optional server-side installation, so other players (with the mod) can see you fly around in the barrel roll way and players even without the mod could the the thrust particle effects. Also having the option on a server to specify if the elytra thrust can be used. I was wondering if this feature could be added because I like using replay mod and spectator mode doesn't show the correct view.

TL;DR:
1. Add optional server-side
2. I'm sorry
3. I want working spectator view (of course only if you have the client mod)
4. Server option to enable/disable thrust

**Comment by enjarai** [comment]
I do plan to add server configs for the thrusting at some point, but letting other players see the custom roll would be very complex. And since this mod is primarily focused on the client side, its currently outside of the scope.

I'll leave this issue open to track server config and shared thrusting particles,

**Comment by enjarai** [comment]
Server-side configurability for thrusting has been implemented in 2.4.0.

**Comment by enjarai** [comment]
Server-side roll syncing has also been implemented as of 2.5.0. This should work with spectator mode.
